### PR TITLE
FINS-6978: Add payroll json format under financial integration

### PR DESCRIPTION
**Pull request by sibidominic**

### CODE REVIEW

**Review by srikanthbabu133 — APPROVED** [review]
+1

File: src/api-reference/financial-integration/v4.financial-documents-schema.markdown [hunk 1]
```diff
@@ -139,6 +139,249 @@ Name|Type|Format|Description
 `cashAdvanceId`|`String`|-|GUUID. Unique system ID assigned to the cash advance. Maximum length 32 characters.
 `cashAdvanceTransactionType`|`Number`|-|Type of transaction: 1 = Issue or return to administrator, 2 = Application, including cash advance return expense within a report, or 3 = System cash advance, from balance carry forward.
 
+## Payroll <a name="scpayroll"></a>
+Name|Type| Format                                                        |Description
+---|---|---------------------------------------------------------------|---
+`report`|`Object`| [Payroll - Report](#scprlrep)                                 |***
+`employee`|`Object`| [Payroll - Employee](#scexpemploy)                            |***
+`cashAdvanceApplication`|`Object`| [Payroll - CashAdvanceUtilizationData](#scprlcaa)             |Currency. Cash advance utilized amount on a given expense report
+`expenseEntry`|`Array`| Array of [Payroll - ExpenseEntry](#scprlexpent)               |***
```
Comment (MarcBell-SAP) [inline src/api-reference/financial-integration/v4.financial-documents-schema.markdown:148]: This should link to the "Payroll - Employee" schema but instead links to the "Expense - Employee" schema. Please fix this.

File: src/api-reference/financial-integration/v4.financial-documents-schema.markdown [hunk 2]
```diff
@@ -139,6 +139,249 @@ Name|Type|Format|Description
...
+`cashAdvanceApplicationAmount`|`Number`|Maximum 23 digits to the left of the decimal; maximum 8 digits to the right of the decimal; with a maximum of 23 digits for the entire field.|Currency. Cash advance utilized amount.
+`cashAdvanceClearingWageType`|`String`|***|***
+`payrollPayIndicator`|`String`|Y/N|This field indicates whether the expense group the report user belongs to is configured to reimburse employees using Payroll.
+`cashAdvanceReturnsAmount`|`Number`|Maximum 23 digits to the left of the decimal; maximum 8 digits to the right of the decimal; with a maximum of 23 digits for the entire field.|Currency. Amount of cash advance returned.
+
+### <a name="scprltnryrsltdtl"></a>Payroll - ItineraryResultDetail
+
+Name|Type|Format|Description
+---|---|---|---
+`itineraryHeader`|`Object`|[Payroll - ItineraryHeader](#scprlItnryHdr)|Header with general information about the itinerary
```
Comment (MarcBell-SAP) [inline src/api-reference/financial-integration/v4.financial-documents-schema.markdown:230]: This is a dead link. Clicking on it does not go anywhere. Please fix.

File: src/api-reference/financial-integration/v4.financial-documents-schema.markdown [hunk 3]
```diff
@@ -139,6 +139,249 @@ Name|Type|Format|Description
...
+`dailyAllowanceDetails`|`Array`| Array of [Payroll - DailyAllowanceDetail](#scprldlyallwnceDtl)|List of summarized and detailed daily travel allowance calculation results of the itinerary.
```
Comment (MarcBell-SAP) [inline src/api-reference/financial-integration/v4.financial-documents-schema.markdown:231]: This is a dead link. Clicking on it does not go anywhere. Please fix.

File: src/api-reference/financial-integration/v4.financial-documents-schema.markdown [hunk 4]
```diff
@@ -139,6 +139,249 @@ Name|Type|Format|Description
...
+`payrollPayIndicator`|`String`|Y/N|This field indicates whether the expense group the report user belongs to is configured to reimburse employees using Payroll.
+`cashAdvanceReturnsAmount`|`Number`|Maximum 23 digits to the left of the decimal; maximum 8 digits to the right of the decimal; with a maximum of 23 digits for the entire field.|Currency. Amount of cash advance returned.
+
+### <a name="scprltnryrsltdtl"></a>Payroll - ItineraryResultDetail
+
+Name|Type|Format|Description
+---|---|---|---
+`itineraryHeader`|`Object`|[Payroll - ItineraryHeader](#scprlItnryHdr)|Header with general information about the itinerary
+`dailyAllowanceDetails`|`Array`| Array of [Payroll - DailyAllowanceDetail](#scprldlyallwnceDtl)|List of summarized and detailed daily travel allowance calculation results of the itinerary. 
+`itineraryResultHeader`|`Object`| [Payroll - ItineraryResultHeader]($scprlitnryRsltHdr)|General data of the itinerary
```
Comment (MarcBell-SAP) [inline src/api-reference/financial-integration/v4.financial-documents-schema.markdown:232]: This link creates a 404 error. See below:
"404 - page not found
The 
6322377c98f77a5a7f86fd2ee548617c63bdac8c

 branch of 
preview.developer.concur.com

 does not contain the path 
src/api-reference/financial-integration/$scprlitnryRsltHdr."

**Review by MarcBell-SAP — CHANGES_REQUESTED** [review]
Please fix the issues I left in the comments. I'll merge this after that's done.

File: src/api-reference/financial-integration/v4.financial-documents-schema.markdown (hunk 1 above)
Comment (MarcBell-SAP) [reply src/api-reference/financial-integration/v4.financial-documents-schema.markdown:148]: Fixed. Tested and it works.

File: src/api-reference/financial-integration/v4.financial-documents-schema.markdown (hunk 4 above)
Comment (MarcBell-SAP) [reply src/api-reference/financial-integration/v4.financial-documents-schema.markdown]: This no longer creates a 404 error. However, it is now a dead link. Please fix.

File: src/api-reference/financial-integration/v4.financial-documents-schema.markdown (hunk 3 above)
Comment (MarcBell-SAP) [reply src/api-reference/financial-integration/v4.financial-documents-schema.markdown:231]: This is still a dead link. Please fix.

File: src/api-reference/financial-integration/v4.financial-documents-schema.markdown (hunk 2 above)
Comment (MarcBell-SAP) [reply src/api-reference/financial-integration/v4.financial-documents-schema.markdown:230]: This is still a dead link. Please fix.

Comment (MarcBell-SAP) [reply src/api-reference/financial-integration/v4.financial-documents-schema.markdown]: Fixed. Tested and works.

File: src/api-reference/financial-integration/v4.financial-documents-schema.markdown (hunk 3 above)
Comment (MarcBell-SAP) [reply src/api-reference/financial-integration/v4.financial-documents-schema.markdown]: Fixed. Tested and works.

File: src/api-reference/financial-integration/v4.financial-documents-schema.markdown (hunk 4 above)
Comment (MarcBell-SAP) [reply src/api-reference/financial-integration/v4.financial-documents-schema.markdown]: Fixed. Tested and works.

**Review by MarcBell-SAP — APPROVED** [review]
:shipit:

**Pull request merged**